src/main/resources/reference.conf

```diff
@@ -652,6 +652,9 @@ resourceTypes = {
       read_spend_report_configuration = {
         description = "read the spend report config (i.e. the BigQuery dataset for spend reports) for this billing-project"
       }
+      read_spend_report = {
```

Comment:
Why is it "read" instead "create"? It seems like potentially one could want both permissions (if we cached the result of running a report), but we plan to only allow billing project owners to both create a report and view it.

Reply:
I went back and forth between `create` and `read`. I ended up choosing `read` over `create` because I felt that was what we were really checking -- whether the user could read data from the BQ dataset. Still, I'm not set on `read` so if you think `create` makes more sense than `read` then we can change it!

Reply:
I thought that we would be generating reports via GQ on the fly. Are you saying that the report data will already be present and we are just returning it? If so, "read" makes sense to me. But if we will be creating anything based on this permission I think that "create" is more descriptive. I'm conditioned to think that there will be both read and write permissions for things.